code/data-tracker/data-tracker.css

```diff
@@ -143,126 +143,195 @@ a.small-button {
+display:block;
+border-radius: 50%;
+line-height: 26px;
+background: white;
+color: #bebebe;
+text-align: center;
+font-weight: bold;
 }
 
 /* This is for the bars */
 .progressbar li{
-  list-style-type: none;
-  float: left;
-  width: 32%;
-  position: relative;
-  text-align: left;
-  margin-left: -5px;
+list-style-type: none;
+float: left;
+width: 32%;
+position: relative;
+text-align: left;
+margin-left: -5px;
 }
 .progressbar li + li {
-  margin-top: unset;
+margin-top: unset;
 }
 
 .progressbar li:after{
-  content: '';
-  position: absolute;
-  width:100%;
-  height: 8px;
-  background: #979797;
-  top: 10px;
-  left: -90%;
-  z-index: -1;
+content: '';
+position: absolute;
+width:100%;
+height: 8px;
+background: #979797;
+top: 10px;
+left: -90%;
+z-index: -1;
 }
 
 /* Hides the first child bar */
 .progressbar li:first-child:after{
-  content: none;
+content: none;
 }
 
 .progressbar li.active + li:after{
-  background: #005EA2;
+background: #005EA2;
 }
 
 .progressbar li.active::before{
-  border: 2px solid white;
-  background: #005EA2;
-  color: white;
+border: 2px solid white;
+background: #005EA2;
+color: white;
 }
 .progressbar li.active{
-  font-weight: bold;
-  color: #005EA2;
+font-weight: bold;
+color: #005EA2;
 }
 .progressbar li.inactive{
-  font-weight: bold;
-  color: #163f6e;
+font-weight: bold;
+color: #163f6e;
 }
 .progressbar li.inactive + li:after{
-  background: #163f6e;
+background: #163f6e;
 }
 
 .progressbar li.inactive::before{
-  border-color: white;
-  background: #163f6e;
-  color: white;
+border-color: white;
+background: #163f6e;
+color: white;
 }
 
 /* For mobile screens step indicator */ 
 @media only screen and (max-width: 600px) {
-  .progressbar li:before{
-    margin-top: -33px;
-    line-height: 18px;
-    width:20px;
-    height:20px;
-    border-width: 1px;
-  }
-  .progressbar li.active::before{
-    border-width: 1px;
-  }
-  
-  .progressbar li, .progressbar li.active, .progressbar li.inactive {
-    font-size: 11px;
-    padding-top: 15px;
-  }
-  
-  .progressbar li:after{
-    border-left: 1px solid white;
-    height: 10px;
-    margin-top: -2em;
-  }
-}
\ No newline at end of file
+.progressbar li:before{
+  margin-top: -33px;
+  line-height: 18px;
+  width:20px;
+  height:20px;
+  border-width: 1px;
+}
+.progressbar li.active::before{
+  border-width: 1px;
+}
+
+.progressbar li, .progressbar li.active, .progressbar li.inactive {
+  font-size: 11px;
+  padding-top: 15px;
+}
+
+.progressbar li:after{
+  border-left: 1px solid white;
+  height: 10px;
+  margin-top: -2em;
+}
+
+}
+/*wraps text in Signals Details>Service Requests - Details field*/
+view_1704 td.field_1446 > * {
+overflow: auto;
+overflow-wrap:
+break-word;
+width: 60px;
+max-height: 100px;
+}
+
+/*********************************************/
+/******** Header Banner Image Styling ********/
+/*********************************************/
+
+/* Base styles for the image */
+img.knHeader__logo-image {
+width: 100% !important;
```

Comment:
The indentation of the class attributes here are different from what's in Knack production. Here they're all left aligned whereas in Knack they're indented.